mathesar/tests/rpc/exceptions/test_error_codes.py

```diff
@@ -1,12 +1,12 @@
-from http.client import CannotSendRequest
+import pytest
 
-from django.core.exceptions import FieldDoesNotExist
 from psycopg.errors import BadCopyFileFormat
-import pytest
+from django.core.exceptions import FieldDoesNotExist
+from mathesar.utils.connections import BadInstallationTarget
+from db.functions.exceptions import UnknownDBFunctionID
 from sqlalchemy.exc import IntegrityError
+from http.client import CannotSendRequest
 
-from db.functions.exceptions import UnknownDBFunctionID
-from mathesar.utils.connections import BadInstallationTarget
 import mathesar.rpc.exceptions.error_codes as err_codes
```

Comment:
Did an automated tool reorder these imports? The new order is pretty non-standard. I'd rather have:

```python
from http.client import CannotSendRequest

from django.core.exceptions import FieldDoesNotExist
from psycopg.errors import BadCopyFileFormat
import pytest
from sqlalchemy.exc import IntegrityError

from db.functions.exceptions import UnknownDBFunctionID
import mathesar.rpc.exceptions.error_codes as err_codes
from mathesar.utils.connections import BadInstallationTarget
```

Reply:
The imports should be grouped into:

1. standard library
2. 3rd party libraries
3. local libraries (our code)

Sorting should be lexicographical by non-keyword within those groups.